module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -710,6 +712,7 @@ protected function formatHoldingItem(
             'location_code' => $locationCode,
             'reserve' => 'TODO',
             'addLink' => true,
+            'bound_with_records' => $boundWithRecords,
```

Comment:
It would probably be a good idea to add bound_with_records support to the Demo driver as well, so people without FOLIO can demonstrate and experiment with this feature. I'll also add a TODO checkbox to update the driver spec when we merge this.

Reply:
To be clear do you want me to do Demo driver work as part of this PR?  Or is that a later thing?

Reply:
If it won't take a lot of time/effort, it might be worth doing now so we can use it during the review process to quickly test edge cases, etc. But if it's going to be a problem, it's not essential!

Reply:
Ok sure I hadn't worked with the Demo driver before, didn't realize it was mostly random data.  Will build this.

Reply:
Done.